network-store-server/src/test/java/com/powsybl/network/store/server/ExtensionHandlerTest.java

```diff
@@ -289,4 +290,38 @@ public void updateExtensionsFromEquipmentsTest() {
         assertEquals(10.0, activePowerControl.getDroop(), 0.1);
         assertEquals(2.0, activePowerControl.getParticipationFactor(), 0.1);
     }
+
+    @Test
+    public void insertNonPersistentExtensionTest() {
+        String equipmentIdA = "idBatteryA";
+
+        OwnerInfo infoBatteryA = new OwnerInfo(
+                equipmentIdA,
+                ResourceType.BATTERY,
+                NETWORK_UUID,
+                Resource.INITIAL_VARIANT_NUM
+        );
+        Map<String, ExtensionAttributes> extensionAttributesMapA = Map.of(
+                "notPersistent", new NonPersistentExtensionAttributes(),
+                "activePowerControl", ActivePowerControlAttributes.builder().droop(6.0).participate(true).participationFactor(1.5).build()
+        );
+        Map<OwnerInfo, Map<String, ExtensionAttributes>> mapA = new HashMap<>();
+        mapA.put(infoBatteryA, extensionAttributesMapA);
+        extensionHandler.insertExtensions(mapA);
```

Comment:
`extensionHandler.insertExtensions(Map.of(infoBattery, extensionAttributes));`

Reply:
done